antarest/study/storage/variantstudy/model/command/create_binding_constraint.py

```diff
@@ -73,15 +74,22 @@ def check_matrix_values(time_step: BindingConstraintFrequency, values: MatrixTyp
 
 class BindingConstraintProperties(BaseModel, extra=Extra.forbid, allow_population_by_field_name=True):
     enabled: bool = True
-    time_step: BindingConstraintFrequency = BindingConstraintFrequency.HOURLY
+    time_step: BindingConstraintFrequency = Field(BindingConstraintFrequency.HOURLY, alias="type")
     operator: BindingConstraintOperator = BindingConstraintOperator.EQUAL
-    comments: t.Optional[str] = ""
-    filter_year_by_year: t.Optional[str] = ""
-    filter_synthesis: t.Optional[str] = ""
+    comments: str = ""
 
 
-class BindingConstraintProperties870(BindingConstraintProperties):
-    group: t.Optional[str] = ""
+class BindingConstraintProperties830(BindingConstraintProperties):
+    filter_year_by_year: str = Field("", alias="filter-year-by-year")
+    filter_synthesis: str = Field("", alias="filter-synthesis")
+
+
+class BindingConstraintProperties870(BindingConstraintProperties830):
+    group: str = "default"
+
+
+class _OptionalProperties(BindingConstraintProperties870, metaclass=AllOptionalMetaclass, use_none=True):
```

Comment:
The `_OptionalProperties` should be public (w/o underscore) since it is used elsewhere.

Reply:
Could you, please, remove the `__all__` constant. Actually, it is noisy and useless.